tests/rptest/tests/partition_balancer_test.py

```diff
@@ -722,6 +722,12 @@ def is_ready_and_stable(s):
             nonlocal ready_appeared_at
             if s["status"] == "ready":
                 if ready_appeared_at is None:
+                    # Disable leader balancer after partition balancer has finished its work,
```

Comment:
probably a dumb question.. why can't we disable leader balancer for the entire duration of the test? (in rp config).

Reply:
Thought about it... I guess it is just makes the test more "natural" if the leader balancer is enabled for most of the test duration and disabled only to make the final assessment.